reporter/src/main/kotlin/reporters/AsciidocTemplateReporter.kt

```diff
@@ -81,32 +84,44 @@ class AsciidocTemplateReporter : Reporter {
     override fun generateReport(input: ReporterInput, outputDir: File, options: Map<String, String>): List<File> {
         val asciidoctorAttributes = AttributesBuilder.attributes()
 
-        options[OPTION_PDF_THEME_PATH]?.let { themePath ->
-            File(themePath).also {
-                require(it.isFile) { "Could not find pdf-theme file at '${it.absolutePath}'." }
-            }
+        // Also see https://github.com/asciidoctor/asciidoctorj/issues/438 for supported backends.
+        val backend = options[OPTION_BACKEND] ?: BACKEND_PDF
+
+        if (backend.equals(BACKEND_PDF, ignoreCase = true)) {
+            options[OPTION_PDF_THEME_PATH]?.let { themePath ->
+                File(themePath).also {
+                    require(it.isFile) { "Could not find pdf-theme file at '${it.absolutePath}'." }
+                }
 
-            asciidoctorAttributes.attribute("pdf-theme", themePath)
+                asciidoctorAttributes.attribute("pdf-theme", themePath)
+            }
         }
 
         val asciidocTempDir = createTempDirectory("$ORT_NAME-asciidoc").toFile()
         val asciidocFiles = templateProcessor.processTemplates(input, asciidocTempDir, options)
 
         val outputFiles = mutableListOf<File>()
 
-        asciidocFiles.forEach { file ->
-            val outputFile = outputDir.resolve("${file.nameWithoutExtension}.pdf")
+        if (backend.equals(ASCIIDOC_FILE_EXTENSION, ignoreCase = true)) {
```

Comment:
I am not sure whether this too intuitive for a user, since `adoc` is technically not a Asciidoctor backend. If this is not a separate option (e.g. a flag), then it least it should be documented that this is possible.

Reply:
Done.